### PR TITLE
Remove jquery-ui from batch 3 of scripts

### LINKED ISSUE #3008: Remove jquery-ui from batch 3 of scripts
Remove jquery-ui from following scripts:
interface/forms/fee_sheet/new.php
interface/forms/group_attendance/new.php
interface/forms/newpatient/common.php
interface/main/messages/messages.php
interface/main/pwd_expires_alert.php
interface/main/tabs/main.php
interface/new/new_comprehensive.php
interface/orders/procedure_provider_edit.php
interface/orders/single_order_results.php
interface/orders/types_edit.php

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3008

### CODE REVIEW

**Comment by tywrenn** [comment]
messages.php actually needs more work since I believe it still uses jQuery-UI autocomplete API's

**Comment by im-Amitto** [comment]
> messages.php actually needs more work since I believe it still uses jQuery-UI autocomplete API's

Thanks for the review, will look into it.

File: interface/orders/single_order_results.php [hunk 1]
```diff
@@ -104,7 +104,7 @@
 ?>
 <html>
 <head>
-    <?php Header::setupHeader(['jquery-ui']); ?>
+    <?php Header::setupHeader([]); ?>
```
Comment (tywrenn) [inline interface/orders/single_order_results.php]: Remove [] brackets

File: interface/forms/fee_sheet/new.php [hunk 2]
```diff
@@ -572,7 +572,7 @@ function echoProductLines()
 ?>
 <html>
 <head>
-<?php Header::setupHeader(['knockout', 'jquery-ui', 'jquery-ui-base']);?>
+<?php Header::setupHeader(['knockout']);?>
```
Comment (tywrenn) [inline interface/forms/fee_sheet/new.php]: JQUI Autocomplete is still present here

File: interface/main/pwd_expires_alert.php [hunk 3]
```diff
@@ -46,7 +46,7 @@
 
 <html>
 <head>
-    <?php Header::setupHeader(["jquery-ui","jquery-ui-darkness"]); ?>
+    <?php Header::setupHeader([]); ?>
```
Comment (tywrenn) [inline interface/main/pwd_expires_alert.php]: Remove [] brackets

File: interface/main/messages/messages.php [hunk 4]
```diff
@@ -786,7 +786,7 @@ function deselectRow(row) {
                         <span class="title"><?php echo xlt('SMS Zone'); ?></span>
                         <br/><br/>
                         <form id="smsForm" class="input-group">
-                            <input id="SMS_patient" type="text" style="margin:0;max-width:100%;" class="form-control" placeholder="<?php echo xla("Patient Name"); ?>" />
+                            <input id="SMS_patient" type="text" style="max-width:100%;" class="form-control m-0" placeholder="<?php echo xla("Patient Name"); ?>" />
```
Comment (tywrenn) [inline interface/main/messages/messages.php]: You can use `mw-100` to replace max width for BS4

File: interface/main/messages/messages.php [hunk 5]
```diff
@@ -536,11 +536,11 @@ class='form-horizontal'
                                                 $body = preg_replace('/(:\d{2}\s\()' . $result['pid'] . '(\sto\s)/', '${1}' . $patientname . '${2}', $body);
                                                 $body = preg_replace('/(\d{4}-\d{2}-\d{2} \d{2}:\d{2}\s\([^)(]+\s)(to)(\s[^)(]+\))/', '${1}' . xl('to{{Destination}}') . '${3}', $body);
                                                 $body =nl2br(text(oeFormatPatientNote($body)));
-                                                echo "<div class='text oe-margin-t-3' style='border: 1px solid var(--gray); padding: 5px;'>" . $body . "</div>";
+                                                echo "<div class='text oe-margin-t-3 p-5' style='border: 1px solid var(--gray);'>" . $body . "</div>";
```
Comment (tywrenn) [inline interface/main/messages/messages.php]: `p-5` is too extreme here. I recommend running a docker environment and going from 1 to 5 to find the right padding for this.

File: interface/forms/newpatient/common.php [hunk 6]
```diff
@@ -483,7 +483,7 @@ function cancelClickedOld() {
                 <legend><?php echo xlt('Link/Add Issues (Injuries/Medical/Allergy) to Current Visit')?></legend>
                     <div id="visit-issues">
                         <div class="form-row px-5">
-                            <div style="padding-bottom: 5px">
+                            <div class="pb-5">
```
Comment (tywrenn) [inline interface/forms/newpatient/common.php]: Also padding too extreme here

**Comment by tywrenn** [comment]
@im-Amitto Please run docker and view the page before adding padding to it because the padding replacements here aren't correct.

File: interface/forms/fee_sheet/new.php [hunk 7]
```diff
@@ -197,8 +197,8 @@ function echoServiceLines()
         } else { // not billed
             if ($institutional) {
                 if ($codetype != 'COPAY' && $codetype != 'ICD10') {
-                    echo "  <td class='billcell'><input type='text' class='revcode' name='bill[" . attr($lino) . "][revenue_code]' " .
-                        "title='" . xla("Revenue Code for this item. Type to search or double click for list") . "' " .
+                    echo "  <td class='billcell'><select style='width:150px' type='text' class='revcode' name='bill[" . attr($lino) . "][revenue_code]' " .
```
Comment (tywrenn) [inline interface/forms/fee_sheet/new.php]: Add closing `</select>`

File: interface/main/messages/messages.php [hunk 8]
```diff
@@ -189,7 +189,7 @@
                     <a href='#' id='recalls-li' class="nav-link font-weight-bold"><?php echo xlt('Recalls'); ?></a>
                 </li>
                 <?php }?>
-                <?php if ($logged_in) { ?>
+                <?php if (1) { ?>
```
Comment (tywrenn) [inline interface/main/messages/messages.php]: Think you didn't mean to put this here

File: interface/main/messages/messages.php [hunk 9]
```diff
@@ -781,16 +781,16 @@ function deselectRow(row) {
         <div class="row oe-display" id="sms-div">
             <div class="col-sm-12">
                 <div class="jumbotron jumbotron-fluid py-3">
-                    <?php if ($logged_in) { ?>
+                    <?php if (1) { ?>
```
Comment (tywrenn) [inline interface/main/messages/messages.php]: Think you didn't mean to put this here

File: interface/main/messages/messages.php [hunk 10]
```diff
@@ -781,16 +781,16 @@ function deselectRow(row) {
...
                     <div class="col-sm-4 col-md-4 col-lg-4">
                         <span class="title"><?php echo xlt('SMS Zone'); ?></span>
                         <br/><br/>
                         <form id="smsForm" class="input-group">
-                            <input id="SMS_patient" type="text" style="max-width:100%;" class="form-control m-0" placeholder="<?php echo xla("Patient Name"); ?>" />
+                            <select id="SMS_patient" type="text" style="width:100%;" class="form-control m-0" placeholder="<?php echo xla("Patient Name"); ?>" />
```
Comment (tywrenn) [inline interface/main/messages/messages.php]: Add closing `</select>` and replace width inline style with `w-100` BS4 class

**Comment by bradymiller** [comment]
hi @im-Amitto , Code looks good and testing well. Bringing this into codebase. Thanks for the code improvements! -brady

**Pull request merged**